Extensions/DateTimeExtensions.cs

```diff
@@ -105,5 +105,8 @@ public static class DateTimeExtensions
 
             return years;
         }
+
+        public static DateTime? ParseToNullableDateTime(this string text) =>

```

Comment:
Inkább stringextension? Nem tudtam dönteni.

Reply:
String a bemenete, szóval StringExtension - átrakom (meg a hasonló bool? kimenetű functiont is).